composer.json

```diff
@@ -16,23 +16,26 @@
             "type": "vcs",
             "url": "https://github.com/sourceability/coding-standard",
             "no-api": true
+        },
+        {
+            "type": "vcs",
+            "url": "git@github.com:webdevel/liquidmetal-php.git"
```

Comment:
> Support for deprecated curly braces for offset access has been removed. 

- https://www.php.net/manual/en/migration80.incompatible.php

Reply:
Even though upstream has already merged my changes (but not tagged for release), removing my fork causes build to fail. For whatever reason, using upstream master branch does not have the latest changes. This could easily be removed when upstream tags and releases. I do not think we should make this a blocker.

Reply:
This is a blocker to merging this PR IMHO.

Any project that will want to use this string-matcher patch, will need to also add `git@github.com:webdevel/liquidmetal-php.git` to the project's composer repositories like you did here. So, might as well just add both `git@github.com:webdevel/liquidmetal-php.git` and `git@github.com:webdevel/string-matcher.git` to your project for now.

Once the need to add `git@github.com:webdevel/liquidmetal-php.git` (along with the `dev-master` version constraint, which is also a blocker imho) is dropped, then this PR can be merged, and a tag created on master.

Reply:
Point taken. I have removed VCS, tagged, published, released my fork and adopted long-term maintenance (https://packagist.org/packages/webdevel/liquidmetal-php) as previous author/maintainer went 6 years without update.
<img width="1435" alt="Screen Shot 2021-02-11 at 10 18 11 AM" src="https://user-images.githubusercontent.com/1288852/107681279-b46c1680-6c53-11eb-9376-6bbeab763ad2.png">
[Latest build is passing](https://travis-ci.com/github/webdevel/string-matcher)